Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -934,11 +931,11 @@ public extension ClusterSystem {
 extension ClusterSystem {
     func receiveInvocation(actor: some DistributedActor, message: InvocationMessage) async {
         guard let shell = self._cluster else {
-            fatalError("FIXME: cluster is inactive already") // FIXME(distributed): fix this and allow returning back an error via the handler from here
+            self.log.error("Cluster has shut down already, yet received message. Message will be dropped: \(message)")
+            return
```

Comment:
just returning is fine, we're ignoring messages when we're already shutting down basically